apps/data/ogr_fields_override.schema.json

```
@@ -0,0 +1,125 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

we could probably use the latest version (not that it brings anything we need):
```suggestion
  "$schema": "https://json-schema.org/draft/2020-12/schema",
```